spec/ttml1.xml

```diff
@@ -3320,6 +3320,11 @@ above if possible.</p>
 <p>The semantics of the above four cited XSL-FO properties are defined by
 by <bibref ref="xsl11"/>, &sect; 7.17.3, 7.16.7, 7.16.12, and 7.16.8, respectively.</p>
 </note>
+<note role="clarification">
```

Comment:
Change as follows:

Add normative text:
 
"If an unnormalized U+0009 Horizontal Tab (HT) character appears in a character information item of a Reduced XML Infoset, then, when presented visually, that HT character may be treated as if it were a single U+0020 (SPACE) character."

then change note to read

"Because the presentation semantics of U+0009 Horizontal Tab (HT) are not fixed, the use of HT in
``<code>#PCDATA</code>`` content is not recommended."

Reply:
The proposed note does not include the observation that _the TAB (U+0009) character is not collapsed when <code>xml:space="default"</code>_, which is was led to this issue in the first place.

Reply:
And, the reason it does not include that observation is that it is incorrect. In fact, ``xml:space='default'`` maps to XSL-FO semantics ``white-space-collapse='true'``, which, for mixed elements, non-line-initial HT are required to be mapped SPACE. It is only the edge case of a mixed element line-initial HT where XSL-FO is unclear on whether it is mapped to SPACE or not, and there is some (indirect) evidence in XSL-FO that it is.

Therefore, there is no question that HT in non-line-initial positions in mixed content are mapped to SPACE (and therefore subject to normal SPACE collapse rules). Only in the case of  line-initial HT in mixed content does a possible ambiguity arise. Furthermore, it would be very odd indeed for an implementation that collapses non-line-initial HTs (which is required) to not also collapse line-initial HTs. In fact, I know of no such implementation.

Even if an implementation did not collapse line-initial HT to SPACE, there are other presentation semantics in XSL-FO that essentially imply that XML white space at a line break are not presented, i.e., do not consume IPD space, except when ``white-space-collapse='false'``.

In TTML2 we clear up  the ambiguity by requiring that line-initial XML whitespace be collapsed when ``white-space-collapse='true'`` applies. We can do this with some confidence that it will not cause an interoperability problem because we know with reasonable certainty that no TTML1 implementation fails to collapse line-initial XML white space.

Reply:
Can we make the same change to clear up the ambiguity in TTML1 Third Edition then?

Reply:
That was my original proposal (above), but @palemieux desired to not mandate collapse of HT as specified in TTML2, but instead (effectively) make it optional. Personally, I have no problem making it mandatory in TTML1 3Ed, which IMO, has very little likelihood of requiring a change in any implementation, since they are already doing the "right thing".

Reply:
What about: _No presentation semantics are specified for the TAB (U+0009) character. Furthermore, the TAB (U+0009) character can generate a glyph area. As a result, the use of the TAB (U+0009) character in <code>#PCDATA</code> content within <code>p</code> and <code>span</code> elements is not recommended._